addons/l10n_id_efaktur_coretax/models/account_move_line.py

```diff
@@ -47,5 +51,6 @@ def _l10n_id_coretax_build_invoice_line_vals(self, vals):
 
         line_val['VAT'] = idr.round(line_val['OtherTaxBase'] * line_val['VATRate'] / 100)
         line_val['STLG'] = idr.round(line_val['STLGRate'] * line_val['OtherTaxBase'] / 100)
+        line_val['TotalDiscount'] = idr.round(line_val['Price'] * line_val['Qty'] * self.discount / 100)
```

Comment:
Why not keep it above? You already have access to all the info needed, it would be clearer.